src/configs/common.yaml

```diff
@@ -0,0 +1,17 @@
+# The &COMMON anchor allows merging these values into other YAML files with <<: *COMMON
+&COMMON
```

Comment:
Interesting! TIL that this is a thing. I worry / wonder whether it's a bit confusing for users and maybe also an anti-pattern somehow? Up for discussion, though. idk...